packages/components/pages/f-mfa/src/components/Mfa.vue

```diff
@@ -2,37 +2,159 @@
+                </form>
+
+                <f-button
+                    :class="$style['c-mfa-need-help-link']"
+                    button-type="link"
+                    data-test-id="mfa-need-help-link"
+                    @click="onShowHelpInfo">
+                    {{ $t('verificationPage.helpModalLinkText') }}
+                </f-button>
+            </div>
+        </f-card>
     </div>
 </template>
 
 <script>
-import { globalisationServices } from '@justeat/f-services';
+import { VueGlobalisationMixin } from '@justeat/f-globalisation';
+
+import FCard from '@justeat/f-card';
+import '@justeat/f-card/dist/f-card.css';
+import FormField from '@justeat/f-form-field';
+import '@justeat/f-form-field/dist/f-form-field.css';
+import FButton from '@justeat/f-button';
+import '@justeat/f-button/dist/f-button.css';
+
+import {
+    BagSurfBgIcon
+} from '@justeat/f-vue-icons';
 import tenantConfigs from '../tenants';
 
+const standardLogTags = ['account-pages', 'mfa'];
+
 export default {
     name: 'VMfa',
 
-    components: {},
+    components: {
+        FCard,
+        FormField,
+        FButton,
+        BagSurfBgIcon
+    },
+
+    mixins: [VueGlobalisationMixin],
 
     props: {
-        locale: {
-            type: String,
-            default: ''
-        },
         smartGatewayBaseUrl: {
             type: String,
             required: true
         }
     },
 
     data () {
-        const locale = globalisationServices.getLocale(tenantConfigs, this.locale, this.$i18n);
-        const localeConfig = tenantConfigs[locale];
-
         return {
-            copy: { ...localeConfig }
+            verificationCode: '',
+            email: '',
+            tenantConfigs,
+            isSubmitting: false,
+            showErrorPage: false
         };
+    },
+
+    computed: {
+        isSubmitButtonDisabled () {
+            return this.verificationCode.length < 1;
+        }
+    },
+
+    mounted () {
+        this.initialise();
+    },
+
+    methods: {
+        /**
+        * TODO
+        */
+        initialise () {
+            try {
+                // TODO - Validate Querystring params - Invalid > raise showErrorPage flag
+                // TODO - Assign to data model
+                this.email = 'email@email.com';
+                this.code = 'someCode123';
+            } catch (error) {
+                this.$log.error('Error validating mfa data', error, standardLogTags);
+            }
+        },
+
+        /**
+        * TODO
+        */
+        async onFormSubmit () {
+            this.isSubmitting = true;
+            try {
+                // TODO
+                // eslint-disable-next-line no-promise-executor-return
+                await new Promise(resolve => setTimeout(resolve, 1000)) // DEBUG - Simulate network delay
+                .then(console.log('DEBUG - onFormSubmit', { verificationCode: this.verificationCode, code: this.code, email: this.email })); // eslint-disable-line no-console
```

Comment:
Not sure if intended, probably a typo here.
Bear in mind this `console.log` is performing before the promise is fulfilled.

Given `await` is in place on previous line, you may opt to just not use `.then` at all on this line.

Reply:
@Ariel-Rodriguez It's just a bit of debug code ... I will be ripped out in next ticket 👍

Reply:
You can just simulate this in your browser btw on the network tab